client/src/Components/About/About.js

```diff
@@ -73,6 +77,7 @@ const About = () => {
     {
       title: t('about.interestsTitle'),
       paragraph: t('about.interestsParagraph'),
+      imageName: 'interests',
```

Comment:
I missed this in my earlier review. It's only a nitpick: could you please move all the `imageName` above the corresponding `title`?